print/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  prefix: 'tw-',
```

Comment:
Ich hätte es jetzt ohne Prefix gemacht, hast du einen spezifischen Grund dafür?

Reply:
Yes: Many of the CSS helper classes overlap between Tailwind and Vuetify (e.g. all the spacing helper classes like mb-4). As long as we need Vuetify for v-calendar, prefix is needed.

If we want to get rid of v-calendar (and Vuetify) anytime in the future, removing the prefix should be pretty straight forward.